docs/basics/101-123-config2.rst

```diff
@@ -328,7 +328,7 @@ Let's try this:
    :workdir: dl-101/DataLad-101
    :language: console
 
-   $ git config --file=.gitmodules --replace-all submodule."recordings/longnow".url "git@github.com:datalad-datasets/longnow-podcasts.git"
+   $ cd .. && git config --file=.gitmodules --replace-all submodule."recordings/longnow".url "git@github.com:datalad-datasets/longnow-podcasts.git"
```

Comment:
I'm not sure I understand the need for this commit, and at it can't be merged as is (it leads to a wrong code execution outcome). 

If you check https://ci.appveyor.com/project/mih/book/builds/51242893 lines 1281 and onward, the output of the code execution changes:

```diff
diff --git a/docs/basics/_examples/DL-101-123-106 b/docs/basics/_examples/DL-101-123-106
index 31e0ba64..b28fdb8d 100644
--- a/docs/basics/_examples/DL-101-123-106
+++ b/docs/basics/_examples/DL-101-123-106
@@ -1 +1 @@
-$ git config --file=.gitmodules --replace-all submodule."recordings/longnow".url "git@github.com:datalad-datasets/longnow-podcasts.git"
+$ cd .. && git config --file=.gitmodules --replace-all submodule."recordings/longnow".url "git@github.com:datalad-datasets/longnow-podcasts.git"
diff --git a/docs/basics/_examples/DL-101-123-107 b/docs/basics/_examples/DL-101-123-107
index d1542e4c..c26aadd6 100644
--- a/docs/basics/_examples/DL-101-123-107
+++ b/docs/basics/_examples/DL-101-123-107
@@ -1,2 +1,2 @@
 $ datalad status
- modified: .gitmodules (file)
+nothing to save, working tree clean
diff --git a/docs/basics/_examples/DL-101-123-108 b/docs/basics/_examples/DL-101-123-108
index 7d959280..0c21f32d 100644
--- a/docs/basics/_examples/DL-101-123-108
+++ b/docs/basics/_examples/DL-101-123-108
@@ -1,12 +1 @@
 $ git diff
-diff --git a/.gitmodules b/.gitmodules
-index 9bc9ee9..11273e1 100644
---- a/.gitmodules
-+++ b/.gitmodules
-@@ -1,5 +1,5 @@
- [submodule "recordings/longnow"]
- 	path = recordings/longnow
--	url = https://github.com/datalad-datasets/longnow-podcasts.git
-+	url = git@github.com:datalad-datasets/longnow-podcasts.git
- 	datalad-id = b3ca2718-8901-11e8-99aa-a0369f7c647e
- 	datalad-url = https://github.com/datalad-datasets/longnow-podcasts.git
diff --git a/docs/basics/_examples/DL-101-123-109 b/docs/basics/_examples/DL-101-123-109
index 20a62e86..1c1b80aa 100644
--- a/docs/basics/_examples/DL-101-123-109
+++ b/docs/basics/_examples/DL-101-123-109
@@ -1,4 +1,4 @@
 $ git checkout .gitmodules
 $ datalad status
-Updated 1 path from the index
+Updated 0 paths from the index
 nothing to save, working tree clean
```


In case this ``cd ..`` was necessary for you while following the book I'd love to understand why. In principle, [the book executes code snippets in these runrecords](https://handbook.datalad.org/en/latest/contributing.html#directives-and-demos) in order, and the  output should reveal (via errors, unexpected output, build failures) if everything happened in the right directory and so forth. That being said, it may well be that there is a code block in between that causes users to change directories or a mis-specified inital directory... When looking through the sources, it all looked kosher to me, but please let me know in case I've overlooked something :)